test/hotspot/jtreg/vmTestbase/nsk/share/jpda/DebugeeProcess.java

```diff
@@ -74,8 +74,8 @@ abstract public class DebugeeProcess {
     /** Argument handler from binder. */
     protected DebugeeArgumentHandler argumentHandler = null;
 
-    /** Need or not to check debuggee process termination at exit. */
-    protected boolean checkTermination = false;
+    /** Need or not to check debuggee process termination. */
+    private boolean checkTermination = true;
```

Comment:
What is the impact of this change to our current testing?

Reply:
The 'checkTermination' is set to true by and jdi/jdwp LocalDebugee implementation. So it should be always set to true initially. 
It is used to check process status and complain and kill debugee if the debugee process hasn't been finished by itself.

I think it could be remove later, but don't want to change any logic now.